super-intel.c

```diff
@@ -6122,7 +6122,8 @@ static int add_to_super_imsm(struct supertype *st, mdu_disk_info_t *dk,
 			pr_err("%s controller supports Multi-Path I/O, Intel (R) VROC does not support multipathing\n",
 			       basename(cntrl_path));
 
-		if (super->orom && !imsm_orom_has_tpv_support(super->orom)) {
+		if (devpath_to_vendor(pci_dev_path) != 0x8086 && super->orom &&
```

Comment:
super->orom should come first. I think compiler will realize that but I don't need to consider compilation if params are passed in right order, so please change it:
```suggestion
		if (super->orom && devpath_to_vendor(pci_dev_path) != 0x8086
```

Reply:
done